CMakeLists.txt

```diff
@@ -705,4 +714,4 @@ else ()
   endif ()
 endif (IS_64BIT_BUILD EQUAL 1)
 
-add_subdirectory(src)
+#add_subdirectory(src)
```

Comment:
Didn't intend to commit that, reducing my own build time while getting PAL working.